code/game/objects/items/weapons/shields.dm

```diff
@@ -41,13 +41,13 @@
 	. = ..()
 	switch(get_block_chance(user))
 		if(0 to 30)
-			to_chat(user, "So heavy... You feel doubtful in your ability to parry with this shield. Maybe if you changed your grip?")
+			to_chat(user, "So heavy... You doubt your ability to parry with this shield until you grow stronger.")
 		if(31 to 45)
-			to_chat(user, "Holding this feels a little clumsy.")
+			to_chat(user, "Holding this feels a little clumsy. Perhaps you should hit the gym.")
 		if(46 to 55)
-			to_chat(user, "With a comfortable grip, you feel condifent in your ability to parry with this shield.")
+			to_chat(user, "A bit hefty, but you feel confident in your ability to parry with this shield.")
 		if(56 to 70)
-			to_chat(user, "The shield feels just right, you feel you can parry anything!")
+			to_chat(user, "The shield weight feels comfortable and maneuverable.")
```

Comment:
60+ robustness required regardless of type of shield, meaning between "Professional" and "Godlike" skill level

Reply:
I felt that saying "Feels like you can parry anything!" implies it's maxed out block chance. There's a tier over that.

Reply:
Yes, and that tier is pretty much a joke: "You spent your entire round grinding stats to get here. Are you happy with your ability to beat roaches now?"

Reply:
This is a fix too far into the other direction I think.
The prior message implied you were maxed out. This one implies you're just at average skill.